pom.xml

```diff
@@ -35,7 +35,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <ddf.version>2.11.5</ddf.version>
+        <ddf.version>2.11.6-SNAPSHOT</ddf.version>
```

Comment:
Ahhh did we forget to move the version forward after the release?

Reply:
`forgot` is maybe not the correct word